perceval/backends/core/jira.py

```diff
@@ -273,8 +273,8 @@ class JiraClient(HttpClient):
 
     :param URL: URL of the JIRA server
     :param project: filter issues by project
-    :param user: JIRA's username
```

Comment:
`username` might be misleading, so it would be great if we can follow the above comment
```
-      :param user: Jira user
+      :param user: Jira user (email)
```